data/manifest_core.json

```diff
@@ -307,113 +307,35 @@
-                }
-              },
-              "na": {
-                "resolution": {
-                  "6m": "",
-                  "2y": "",
-                  "3y": "",
-                  "6y": "",
-                  "12y": "",
-                  "default": "2y"
-                }
-              },
-              "default": "ha"
-            }
-          },
-          "h1n1pdm": {
-            "segment": {
-              "ha": {
-                "resolution": {
-                  "6m": "",
-                  "2y": "",
-                  "3y": "",
-                  "6y": "",
-                  "12y": "",
-                  "pandemic": "",
-                  "default": "2y"
-                }
-              },
-              "na": {
-                "resolution": {
-                  "6m": "",
-                  "2y": "",
-                  "3y": "",
-                  "6y": "",
-                  "12y": "",
-                  "pandemic": "",
-                  "default": "2y"
-                }
-              },
-              "default": "ha"
+      "hmpv": {
+        "type": {
+          "all": {
+            "region":{
+              "genome": "",
+              "G": "",
+              "F": "",
+              "default":"genome"
             }
           },
-          "vic": {
-            "segment": {
-              "ha": {
-                "resolution": {
-                  "6m": "",
-                  "2y": "",
-                  "3y": "",
-                  "6y": "",
-                  "12y": "",
-                  "default": "2y"
-                }
-              },
-              "na": {
-                "resolution": {
-                  "6m": "",
-                  "2y": "",
-                  "3y": "",
-                  "6y": "",
-                  "12y": "",
-                  "default": "2y"
-                }
-              },
-              "default": "ha"
+          "a": {
+            "region":{
+              "genome": "",
+              "G": "",
+              "F": "",
+              "default":"genome"
             }
           },
-          "yam": {
-            "segment": {
-              "ha": {
-                "resolution": {
-                  "6m": "",
-                  "2y": "",
-                  "3y": "",
-                  "6y": "",
-                  "12y": "",
-                  "default": "2y"
-                }
-              },
-              "na": {
-                "resolution": {
-                  "6m": "",
-                  "2y": "",
-                  "3y": "",
-                  "6y": "",
-                  "12y": "",
-                  "default": "2y"
-                }
-              },
-              "default": "ha"
+          "b": {
+            "region":{
+              "genome": "",
+              "G": "",
+              "F": "",
+              "default":"genome"
             }
           },
-          "default": "h3n2"
+          "default": "all"
         }
-      },
+      },      
```

Comment:
Unnecessary whitespace. I'll remove and get this merged.

Reply:
Thanks for the catch! I thought my IDE would have stripped this.